maven-model/src/test/java/org/apache/maven/model/OrganizationTest.java

```diff
@@ -53,4 +53,36 @@ public void testToStringNullSafe()
         assertNotNull( new Organization().toString() );
     }
 
+
+    public void testToStringNotNonsense11()
+    {
+        Organization org = new Organization();
+        org.setName( "Testing Maven Unit" );
+        org.setUrl( "https://maven.localdomain" );
+
+        assertEquals( "name = 'Testing Maven Unit'\nurl = 'https://maven.localdomain'", org.toString() );
```

Comment:
That looks wrong.  The `toString()` result should at least have the simple class name in it.
Same for `MailingList`, `Scm` and others...